### ISSUE TITLE
Using dot notation on qase.suites() doesn't create nested suites in qase repository 

### ISSUE THREAD

**Issue by dsetiawan230294** [qase-tms/qase-python#296]
Snippet's from decorators :
```
@staticmethod
    def suite(title: str, description: str = None):
        """
        >>> @qase.suite("Sign up")
        >>> def test_example():
        >>>     pass

        :param title: a string with suite name. You can use dot notation to create nested suites.
        :param description: a string with suite description
        :return: pytest.mark instance
        """
        return pytest.mark.qase_suite(title=title, description=description)
```

How I use it : 
```
from qase.pytest import qase

@qase.suite("First Suites.Second Suites.Third Suites")
@qase.title("This is Example Title")
def test_example():
    assert True
```

Instead of creating suites with structure like this : 
```
.
└── Root Suites/
    └── First Suites/
        └── Second Suites/
            └── Third Suites
```

it's create new suites with name exactly like this `First Suites.Second Suites.Third Suites`

**Comment by gibiw** [comment]
@dsetiawan230294 Hi! Thank you for reporting the issue. It has been fixed in version 6.1.8.

**Comment by dsetiawan230294** [comment]
@gibiw Hi ! Confirmed it's fixed in version 6.1.8, thanks !
